Graphs/TreeGraph/index.js

```diff
@@ -1,37 +1,27 @@
 
 import PropTypes from 'prop-types';
 import React from "react";
-import * as d3 from "d3";
 import _ from 'lodash';
 import AbstractGraph from "../AbstractGraph";
 import { properties } from "./default.config";
-import { List } from 'immutable';
 import './styles.css'
-
-const diagonal = (s, d) => {
-    // Creates a curved (diagonal) path from parent to the child nodes
-    const path = `M ${s.y} ${s.x}
-    C ${(s.y + d.y) / 2} ${s.x},
-        ${(s.y + d.y) / 2} ${d.x},
-        ${d.y} ${d.x}`
-
-    return path
-}
-
-// TODO: Make this dynamic as per graph config
-const PAGINATION = 2;
+import { parseSrc } from '@/lib/ui-components/utils.js'
```

Comment:
we cannot import this from the application! This is supposed to be a lib! If passing the base64 encoded image then you will not need this.